Packs/CoreAlertFields/IncidentFields/incidentfield-osparentname.json

```diff
@@ -27,5 +27,16 @@
     "useAsKpi": false,
     "version": -1,
     "fromVersion": "6.5.0",
+<<<<<<< HEAD
+	"x2_fields": "os_actor_process_image_name",
+    "Aliases": [
+        {
+          "cliName": "parentprocess",
```

Comment:
Should go to CGO name (causality_actor_process_image_name)

Reply:
fixed